lib/vector-core/buffers/src/disk/mod.rs

```diff
@@ -19,12 +19,12 @@ pub enum DataDirError {
     NotFound { data_dir: PathBuf },
     #[snafu(display("The configured data_dir {:?} is not writable by the vector process, please ensure vector can write to that directory", data_dir))]
     NotWritable { data_dir: PathBuf },
-    #[snafu(display("Unable to look up data_dir {:?}", data_dir))]
+    #[snafu(display("Unable to look up data_dir {:?} | {:?}", data_dir, source))]
```

Comment:
I think we would normally write this with a colon:
```suggestion
    #[snafu(display("Unable to look up data_dir {:?}: {:?}", data_dir, source))]
```

Reply:
Resolved in 9e3f59c